client/netmap.go

```diff
@@ -41,10 +41,8 @@ func (x ResEndpointInfo) NodeInfo() netmap.NodeInfo {
 //
 // Method can be used as a health check to see if node is alive and responds to requests.
 //
-// Any client's internal or transport errors are returned as `error`.
-// If PrmInit.ResolveNeoFSFailures has been called, unsuccessful
-// NeoFS status codes are returned as `error`, otherwise, are included
-// in the returned result structure.
+// Any client's internal or transport errors are returned as `error`,
```

Comment:
is that (and some more) method special? (the commit changes _that_ comment in a different way compared to other methods)

Reply:
No, it isn't. In general the changes should be similar - remove old instructions and add new